packages/odyssey-react/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -0,0 +1,158 @@
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { useCallback } from "react";
+import { NativeSelectOption } from "./NativeSelectOption";
+import { NativeSelectOptionGroup } from "./NativeSelectOptionGroup";
+import { Field } from "../Field";
+import { ChevronDownIcon } from "../Icon";
+import { withTheme } from "@okta/odyssey-react-theme";
+import { forwardRefWithStatics, useOmit, useOid } from "../../utils";
+import { theme } from "./NativeSelect.theme";
+import styles from "./NativeSelect.module.scss";
+
+import type {
+  ComponentPropsWithRef,
+  ReactNode,
+  ChangeEvent,
+  FocusEventHandler,
+} from "react";
+import type { CommonFieldProps } from "../Field/types";
+
+export interface NativeSelectProps
+  extends CommonFieldProps,
+    Omit<ComponentPropsWithRef<"select">, "style" | "className" | "multiple"> {
+  /**
+   * Select options
+   */
+  children: ReactNode;
+
+  /**
+   * The underlying select element id attribute. Automatically generated if not provided
+   */
+  id?: string;
+  /**
+   * The underlying select element name attribute for the group
+   */
+  name: string;
+
+  /**
+   * The underlying select element disabled attribute for the group
+   * @default false
+   */
+  disabled?: boolean;
+
+  /**
+   * The selected option value attribute for a controlled group.
+   */
+  value?: string;
+
+  /**
+   * Callback executed when the select fires a blur event
+   * @param {Object} event the event object
+   */
+  onBlur?: FocusEventHandler<HTMLSelectElement>;
+
+  /**
+   * Callback executed when the select fires a change event
+   * @param {Object} event the event object
+   * @param {string} value the string value of the select
+   */
+  onChange?: (event?: ChangeEvent<HTMLSelectElement>, value?: string) => void;
+
+  /**
+   * Callback executed when the select fires a focus event
+   * @param {Object} event the event object
+   */
+  onFocus?: FocusEventHandler<HTMLSelectElement>;
+}
+
+type Statics = {
+  Option: typeof NativeSelectOption;
+  OptionGroup: typeof NativeSelectOptionGroup;
+};
+
+/**
+ * Native select input to choose one option from a list
+ */
+export const NativeSelect = withTheme(
+  theme,
+  styles
+)(
+  forwardRefWithStatics<HTMLSelectElement, NativeSelectProps, Statics>(
+    (props, ref) => {
+      const {
+        children,
+        disabled = false,
+        error,
+        hint,
+        id,
+        label,
+        name,
+        onBlur,
+        onChange,
+        onFocus,
+        optionalLabel,
+        required,
+        value,
+        ...rest
+      } = props;
+      const omitProps = useOmit(rest);
```

Comment:
```suggestion
      const omitProps = useOmit(rest, "multiple");
```

to avoid setting the multiple attribute here if we don't support it

Reply:
"multiple" seems to be just esoteric enough to fall outside whatever defines the second argument type. Returns the following error: 
```
Argument of type '"multiple"' is not assignable to parameter of type '"ref" | "form" | "slot" | "title" | "key" | "autoComplete" | "autoFocus" | "size" | "defaultChecked" | "defaultValue" | "suppressContentEditableWarning" | "suppressHydrationWarning" | ... 241 more ... | "labelHidden"'.
```

Reply:
hmmmm this is because we've already omitted the type for `multiple `above which the generic within `useOmit` is working against. However this code operates at run time and provides another level of protection so we may need to work around the compiler here:

```
const omitProps = useOmit<typeof rest & { multiple: "string" }>(rest, "multiple");
```

something like this?

Reply:
the generic is looking for a second argument, this makes the compiler happy
```
type RestProps = typeof rest & { multiple?: string };
type RestPropKeys = (keyof RestProps)[];
const omitProps = useOmit<RestProps, RestPropKeys>(rest, "multiple");
```
updated in f1d952b